src/components/Notifications.vue

```diff
@@ -0,0 +1,103 @@
+    <oc-drop toggle="#notification-bell" boundary="#oc-topbar">
+      <div v-for="(el, index) in activeNotifications" :key="el.notification_id" slot="special" class="uk-card uk-card-default">
+        <a v-if="el.link!==''" :href="el.link" target="_blank" class="uk-link-reset">
+          <div class="uk-card-header" :class="{ 'border-none': lastNotificationIndex === index }">
+            <div><b>{{ el.subject }}</b></div>
+            <div v-if="el.message" class="uk-text-small">{{ el.message }}</div>
+            <div>
+              <a class="uk-text-small link" href=""
+                 @click.prevent.once="deleteNotification({client: $client, notification: el.notification_id})">
+                Mark as read
+              </a>
+            </div>
+            <div v-if="el.actions.length!==0" class="uk-text-small">
+              <a v-for="action in sortedActions(el.actions)" :key="action.label" :class="{ 'uk-text-bold': action.primary }"
+                 class="link" href="" @click.prevent="executeRequest(el.app, action.link, action.type)">
+                {{ action.label }}
+              </a>
+            </div>
+          </div>
+        </a>
+        <div v-else class="uk-card-header" :class="{ 'border-none': lastNotificationIndex === index }">
+          <div><b>{{ el.subject }}</b></div>
+          <div v-if="el.message" class="uk-text-small">{{ el.message }}</div>
+          <div>
+            <a class="uk-text-small link" href=""
+               @click.prevent.once="deleteNotification({client: $client, notification: el.notification_id})">
+              Mark as read
+            </a>
+            <div v-if="el.actions.length!==0" class="uk-text-small">
+              <a v-for="action in sortedActions(el.actions)" :key="action.label" :class="{ 'uk-text-bold': action.primary }"
+                 class="link" href="" @click.prevent="executeRequest(el.app, action.link, action.type)">
+                {{ action.label }}
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </oc-drop>
+  </div>
+</template>
+<script>
+import { mapGetters, mapActions } from 'vuex'
+
+export default {
+  data: function () {
+    return {
+      data: []
+    }
+  },
+  methods: {
+    ...mapActions(['deleteNotification']),
+    sortedActions (actions) {
+      let result = []
+      actions.forEach((a) => {
+        if (a.primary) {
+          result.push(a)
+        }
+      })
+      actions.forEach((a) => {
+        if (!a.primary) {
+          result.push(a)
+        }
+      })
+      return result
+    },
+    executeRequest (app, link, type) {
+      this.$client.requests.ocs({
+        service: 'apps/' + app,
+        action: link,
+        method: type
+      })
+    }
+  },
+  computed: {
+    ...mapGetters(['activeNotifications']),
+    lastNotificationIndex () {
+      return this.activeNotifications.length - 1
+    }
+  }
+}
+</script>
+<style scoped>
```

Comment:
generally speaking we don't want to have css in phoenix ... this needs to go into ods .... somehow... 
please chat with @felixheidecke about this - THX